lib/alerts_viewer_web/router.ex

```diff
@@ -18,17 +19,23 @@ defmodule AlertsViewerWeb.Router do
     get("/_health", HealthController, :index)
   end
 
-  scope "/", AlertsViewerWeb do
-    pipe_through(:browser)
+  scope "/_flags", Laboratory do
+    forward "/", Router
```

Comment:
```suggestion
    pipe_through(:browser)
    
    forward "/", Router
```
question: Should this be piped through the `browser` pipeline?